Makefile

```diff
@@ -628,6 +628,7 @@ $(STATUS_CLIENT_APPIMAGE): nim_status_client $(APPIMAGE_TOOL) nim-status.desktop
 		-no-copy-copyright-files \
 		-qmldir=ui -qmlimport=$(QT5_QMLDIR) \
 		-bundle-non-qt-libs \
+		-exclude-libs="libnss3.so,libnssutil3.so,libgmodule-2.0.so.0,libgthread-2.0.so.0" \
```

Comment:
@yakimant This seems to fix the app crash on initialisation. But I still have issues with missing resources for QtWebEngineView.